data/bss-factory-sets.json

```diff
@@ -0,0 +1,5689 @@
+      },
+      {
+        "species": "azumarill",
+        "weight": 5,
+        "moves": [["bellydrum"], ["aquajet"], ["playrough"], ["terablast"]],
+        "item": ["sitrusberry"],
+        "nature": "adamant",
+        "evs": [244, 252, 0, 0, 0, 12],
+        "teratype": ["fire"],
+        "wantsTera": true,
+        "ability": "hugepower"
+      },
+      {
+        "species": "azumarill",
+        "weight": 5,
+        "moves": [
+          ["substitute"],
+          ["encore"],
+          ["playrough"],
+          ["aquajet", "liquidation"]
+        ],
+        "item": ["leftovers"],
+        "nature": "adamant",
+        "evs": [252, 252, 0, 0, 0, 4],
+        "teratype": ["water"],
+        "ability": "hugepower"
+      }
+    ]
+  },
+  "ironvaliant": {
+    "weight": 5,
+    "sets": [
+      {
+        "species": "ironvaliant",
+        "weight": 35,
+        "moves": [
+          ["spiritbreak"],
+          ["reflect", "encore"],
+          ["closecombat"],
+          ["destinybond", "knockoff"]
+        ],
+        "item": ["boosterenergy"],
+        "nature": "jolly",
+        "evs": [92, 204, 4, 0, 4, 204],
+        "teratype": ["steel", "ghost"],
+        "ability": "quarkdrive"
+      },
+      {
+        "species": "ironvaliant",
+        "weight": 25,
+        "moves": [
+          ["moonblast"],
+          ["encore", "shadowball", "shadowsneak"],
+          ["closecombat", "aurasphere", "shadowball"],
+          ["destinybond"]
+        ],
+        "item": ["focussash", "boosterenergy"],
+        "nature": "naive",
+        "evs": [0, 4, 0, 252, 0, 252],
+        "teratype": ["ghost", "fairy", "steel"],
+        "ability": "quarkdrive"
+      },
+      {
+        "species": "ironvaliant",
+        "weight": 20,
+        "moves": [
+          ["moonblast"],
+          ["closecombat"],
+          ["encore", "shadowball"],
+          ["psyshock", "psychic", "shadowball", "thunderbolt"]
+        ],
+        "item": ["lifeorb", "boosterenergy"],
+        "nature": "naive",
+        "evs": [0, 4, 0, 252, 0, 252],
+        "teratype": ["ghost", "fairy", "steel", "electric"],
+        "ability": "quarkdrive"
+      },
+      {
+        "species": "ironvaliant",
+        "weight": 20,
+        "moves": [
+          ["swordsdance"],
+          ["closecombat", "encore"],
+          ["spiritbreak"],
+          ["shadowsneak", "knockoff"]
+        ],
+        "item": ["focussash", "boosterenergy"],
+        "nature": "jolly",
+        "evs": [0, 252, 4, 0, 0, 252],
+        "teratype": ["ghost", "steel", "fairy"],
+        "ability": "quarkdrive"
+      }
+    ]
+  },
+  "kingambit": {
+    "weight": 5,
+    "sets": [
+      {
+        "species": "kingambit",
+        "weight": 25,
+        "moves": [
+          ["kowtowcleave"],
+          ["suckerpunch"],
+          ["ironhead"],
+          ["guillotine"]
+        ],
+        "item": ["assaultvest"],
+        "nature": "adamant",
+        "evs": [252, 252, 0, 0, 0, 4],
```

Comment:
Please don't put the 4 in Speed on max/max sets in randoms. Move it elsewhere. Also applies for other Kingambits.